provider/resource_workflow_task_update_opsgenie_incident.go

```diff
@@ -84,7 +84,7 @@ func resourceWorkflowTaskUpdateOpsgenieIncident() *schema.Resource {
 							Description: ". Value must be one of `P1`, `P2`, `P3`, `P4`, `auto`.",
 							Type: schema.TypeString,
 							Optional: true,
-							Default: "P1",
+							Default: nil,
```

Comment:
This doesn't match the description

Description: Value must be one of `P1`, `P2`, `P3`, `P4`, `auto`.